### PR TITLE
[DEV-1277] Update profile page copy

### PR DESCRIPTION
**Pull request by jeremygordillo**
#### List of Changes

Updated some labels of profile section 
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .changeset/fifty-geese-hammer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": minor
```
Comment (kin0992) [inline .changeset/fifty-geese-hammer.md:2]: Shouldn't be a patch?

Comment (kin0992) [reply .changeset/fifty-geese-hammer.md:2]: If you think this is a patch, then I'd change the "Types of changes" in the PR description to "Bug fix".
Then, please fill the template with missing information.

File: apps/nextjs-website/src/messages/it.json [hunk 2]
```diff
@@ -31,9 +31,9 @@
         },
         "hello": "Ciao,",
```
Comment (marcobottaro) [inline apps/nextjs-website/src/messages/it.json:32]: I think that this is a "ghost" translation string and you should remove it

**Review by marcobottaro — CHANGES_REQUESTED** [review]
Remove ghost

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**